Makefile

```diff
@@ -20,6 +21,15 @@ compile:
 	mkdir -p build && cd build && $(CMAKE_COMMAND) ${CMAKE_FLAGS} ..
 	cd build && $(MAKE)
 
+check_tclshrc:
```

Comment:
Why do you need this ~/.<file> ? We should not force/need any file in user home directory.

Reply:
On invoking TCL shell, it always first look for `.tclshrc` file in user home directory and load/source everything written in it. 
I am using it so that the shared library of OpenFPGA loaded (if certain condition meet) as user invoke the TCL shell. The other way of loading shared library is to invoke the TCL shell and then use `load` command to load the shared library. 
Let me know the other ways to load the shared library.

Reply:
You are assuming that this will be loaded in tclsh, it is a use case, but most likely not THE use case.
The shared object is going to be loaded in a Tcl interpreter like the one present in FOEDAG https://github.com/os-fpga/FOEDAG. The user will not have to have any .tclshrc nor load the .so manually. It will be automatic.